### DISCUSSION TITLE
How to put colums/text/buttons onto an image and not below/above it?

### DISCUSSION THREAD

**Discussion by gelsas** [GrapesJS/grapesjs#4307] [Q&A]
Hi how do I put colums/text/buttons onto an image ?

I tried to make it work in the Demo: https://grapesjs.com/demo.html

But I can not drag and drop anything onto the image. just below or above it

**Comment by artf** [comment]
I guess you have to apply BG image style and not the image component

**Reply by gelsas** [reply]
Could you elaborate on that I am not sure I fully understand

**Reply by lexoyo** [reply]
@gelsas instead of 

- image
- text
- button

do this:

container with a background image
- text
- button

use this to add an image to a container: 
![Screenshot from 2022-05-10 12-32-10](https://user-images.githubusercontent.com/715377/167609221-9c74233f-76fe-46ab-8089-e366bfd5a957.png)
